pkg/fluentbitconfig/fluentbit-config.go

```diff
@@ -10,12 +10,14 @@ type (
 	Config struct {
 		Service  Service
 		Input    []Input
+		Filter   []Filter
 		Output   []Output
 		Includes []Include
 	}
 
 	Service map[string]string
 	Input   map[string]string
+	Filter  map[string]string
```

Comment:
As this can be passed directly by a user we maybe need to consider some validations on key and value of the map, e.g. it should not contain new line characters or exceed a certain length. What do you think?